Add CD logo and minor fixes

bdfinst opened the pull request:
Added the logo Dana created with related favicon 
Fixed main page navigation problem


![minimumCD logo-hex](https://user-images.githubusercontent.com/8195127/194544050-0675e3a4-defe-4469-b674-10fb24cbd4f1.png)

tracybannon commented:
Not too shabby RE:  the new logo!

tracybannon reviewed (commented):
What was the actual issue with Navigation, @bdfinst ??

bdfinst commented:
When I moved the signatures to a data file with a partial code, I moved the section header with it. That broke the link to the signature anchor. I had to move the section header back to the main page to fix the link.
